documentation/contributing/introduction.adoc

```diff
@@ -9,13 +9,52 @@ This guide is meant to assist contributors to the Strimzi documentation.
 
 == Project layout
 
+Strimzi uses git to manage repositories.
+
+https://github.com/strimzi/strimzi-kafka-operator[`strimzi-kafka-operator`^] (GitHub):: The public GitHub repo hosts all of the Strimzi code and documentation.
+https://github.com/strimzi/strimzi-kafka-operator/tree/master/documentation[`strimzi-kafka-operator/documentation`^] (GitHub):: The _documentation_ folder is split into _category_ folders to manage the content.
+
+Category folders contain files related to Strimzi guides (Deploying, Quickstart, Overview, Using), and the files that provide the content for one or more of these guides – _assemblies_ and _modules_.
+Assemblies, which usually encapsulate a feature or process, bring the related content contained in modules together.
```

Comment:
An analogy would be helpful here. 

An assembly is like a sub-section or chapter in a printed book. For example, a book on _Event-driven architectures_ might have a chapter on _Stream processing_, which would be an assembly.

Reply:
An assembly is like a sub-section or chapter in a book.
For example, the _Deploying Strimzi_ guide has a chapter called _Verifying the {ProductName} deployment_, which is contained in its own assembly.